mono/utils/mono-threads-windows.c

```diff
@@ -289,19 +289,20 @@ mono_threads_suspend_begin_async_resume (MonoThreadInfo *info)
 		info->async_target = NULL;
 		info->user_data = NULL;
 
-		context.ContextFlags = CONTEXT_INTEGER | CONTEXT_CONTROL;
+		context.ContextFlags = CONTEXT_INTEGER | CONTEXT_FLOATING_POINT | CONTEXT_CONTROL;
 
 		if (!GetThreadContext (handle, &context)) {
 			THREADS_SUSPEND_DEBUG ("RESUME FAILED (GetThreadContext), id=%p, err=%u\n", GUINT_TO_POINTER (mono_thread_info_get_tid (info)), GetLastError ());
 			return FALSE;
 		}
 
 		g_assert (context.ContextFlags & CONTEXT_INTEGER);
+		g_assert (context.ContextFlags & CONTEXT_FLOATING_POINT);
 		g_assert (context.ContextFlags & CONTEXT_CONTROL);
 
```

Comment:
No need for these 3 asserts.

Reply:
I've never seen these to be outputs, only inputs. But I'll check.

Reply:
Same as above, guess we could eliminate the checks.